config/kubernetes-incubator/org.yaml

```diff
@@ -582,6 +582,8 @@ teams:
     - pwittrock
     members:
     - sarahnovotny
+    - tengqm
+    - jimangel
```

Comment:
Feel free to add @jimangel as a member to the kubernetes-incubator org here: https://github.com/kubernetes/org/blob/master/config/kubernetes-incubator/org.yaml#L17

He is already a member of the Kubernetes org and is implicitly eligible for membership to kubernetes-incubator.

Reply:
Created a PR for that: https://github.com/kubernetes/org/pull/654 - thanks!